example/lib/main.dart

```diff
@@ -7,13 +7,19 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:time/time.dart';
 import 'package:timetable/timetable.dart';
+// import 'package:timezone/data/latest.dart' as tz;
 
 // ignore: unused_import
 import 'positioning_demo.dart';
 import 'utils.dart';
 
 Future<void> main() async {
   initDebugOverlay();
+  // Please ensure the time zone database is initialized
+  // when you need to display current time indicator with specific time zone.
+  //
+  // More information: [https://pub.dev/packages/timezone]
+  // tz.initializeTimeZones();
```

Comment:
I would also like to provide an API like `DateTimeTimetable.initializeTimeZones()` to make it easier to use.